mne/export/_edf.py

```diff
@@ -53,7 +58,20 @@ def _export_raw(fname, raw, physical_range):
         stim_index = np.atleast_1d(stim_index.squeeze()).tolist()
         drop_chs.extend([raw.ch_names[idx] for idx in stim_index])
 
+    # add warning if any channel types are not voltage based
+    # note: we can write these other channels, such as 'misc'
+    # but they just won't work because EDF doesn't support
+    # the idea of a 'misc' channel
```

Comment:
This comment should be improved. It doesn't really explain this issue (`but they just won't work`) and it somewhat contradicts the warning message (`...set their channel type to "misc"`). Can you clearly explain what users are expected to do?

Reply:
Added some additional clarification.